lib/report_formatter/c3.rb

```diff
@@ -91,7 +91,8 @@ def build_document_header
         unless mri.graph[:type] == 'Donut' || mri.graph[:type] == 'Pie'
           mri.chart[:legend] = {:position => 'bottom'}
         end
-        format, options = javascript_format(mri.graph[:columns][0], nil)
+        column =  mri.performance&.dig(:group_by_category) ? mri.graph[:columns][0].split(/_+/)[0..-2].join('_') : mri.graph[:columns][0]
```

Comment:
Don't use `&.` for the same reason as here: https://github.com/ManageIQ/manageiq/pull/10663